addons/udes_stock/data/locations.xml

```diff
@@ -33,5 +33,12 @@
       <field name="active" eval="True"/>
     </record>
 
+    <!-- Construct "UPL/Stock investigation" location -->
+    <record id="location_stock_investigation" model="stock.location">
+      <field name="name">Stock Investigation</field>
+      <field name="comment">Stock Investigation</field>
```

Comment:
Nit: It feels like the comment can be more detailed than name. Example:
Location used when , i am not good with English

Reply:
This is done. There's not really much to say about it.